jigna/js/build.js

```diff
@@ -68,3 +69,37 @@ concatenate({
         'jigna'
     ]
 });
+
+// jigna-vue.js
+concatenate({
```

Comment:
We do not start the angular app. An angular app is not started unless you a) either explicitly call `angular.bootstrap` or b) use the `ng-app` directive in the HTML. I can agree to distributing two files - jigna.js and jigna-vue.js though, but maybe just add a comment before the first `concatenate` call that jigna.js is jigna with angular.